x/concentrated-liquidity/incentives_test.go

```diff
@@ -2906,13 +2906,13 @@ func (s *KeeperTestSuite) TestQueryAndCollectIncentives() {
 			// Set up invalid pool ID for error-catching case(s)
 			sutPoolId := validPoolId
 			if tc.isInvalidPoolIdGiven {
-				sutPoolId = sutPoolId + 1
+				sutPoolId = sutPoolId + 1 //nolint:staticcheck,ineffassign // this is an ineffectual asisgnment, but i worry about disturbing nature's delicate balance by changing this odd conditional here.
```

Comment:
yeah, agreed that this can be better designed, excluding the comment for now though
```suggestion
				sutPoolId = sutPoolId + 1 //nolint:staticcheck,ineffassign
```